ts/features/idpay/payment/screens/IDPayPaymentAuthorizationScreen.tsx

```diff
@@ -59,39 +71,196 @@ const IDPayPaymentAuthorizationScreen = () => {
+          }}
+        />
+      </SafeAreaView>
     </BaseScreenComponent>
   );
 };
 
-export type { IDPayPaymentAuthorizationScreenRouteParams };
+const AuthorizationScreenContent = ({
+  data
+}: {
+  data: NonNullable<AuthPaymentResponseDTO>;
+}) => {
+  const amountCents = formatNumberCurrency(data.amountCents / 100);
+  const reward = pipe(
+    data.reward,
+    O.fromNullable,
+    O.fold(
+      () => "-",
+      data => formatNumberCurrency(data / 100)
+    )
+  );
+  const businessName = data.businessName || "-";
+  const initiativeName = data.initiativeName || "-";
+  const date = formatDateOrDefault(data.trxDate);
+  return (
+    <>
+      <Divider />
+      <VSpacer size={16} />
+      <View style={[IOStyles.rowSpaceBetween, IOStyles.alignCenter]}>
+        <NewH6>{I18n.t("idpay.payment.authorization.toAuth")}</NewH6>
+        <H1>{reward}</H1>
+      </View>
+
+      <VSpacer size={16} />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.amount")}
+        value={amountCents}
+        accessibilityLabel={I18n.t("idpay.payment.authorization.amount")}
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.businessName")}
+        value={businessName}
+        accessibilityLabel={I18n.t("idpay.payment.authorization.businessName")}
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.dateTime")}
+        value={date}
+        accessibilityLabel={I18n.t("idpay.payment.authorization.dateTime")}
+      />
+
+      <VSpacer size={24} />
+      {/* TODO:: will be removed in favor of LIST_GROUP_HEADING in future updates */}
+      <View style={[IOStyles.row, IOStyles.alignCenter]}>
+        <Icon name="initiatives" size={24} color="bluegrey" />
+        <HSpacer size={16} />
+        <H3
+          color="bluegrey"
+          style={{
+            // this should not happen, but the current typography adds 4
+            // to paddingBottom because of line height
+            // so we add 4 to paddingTop to compensate, else the text would not be centered
+            // (this was temporarily approved by @dmnplb)
+            paddingTop: 4
+          }}
+        >
+          {I18n.t("idpay.payment.authorization.infoDivider")}
+        </H3>
+      </View>
+      <VSpacer size={16} />
+
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.initiativeName")}
+        value={initiativeName}
+        accessibilityLabel={I18n.t(
+          "idpay.payment.authorization.initiativeName"
+        )}
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.availableAmount")}
+        value={"-"}
+        accessibilityLabel={I18n.t(
+          "idpay.payment.authorization.availableAmount"
+        )}
+      />
+    </>
+  );
+};
+
+const SmallSkeleton = () => <Skeleton width={178} height={16} radius={8} />;
```

Comment:
https://github.com/pagopa/io-app/assets/6160324/27fc9eb6-844b-4cbb-9512-f16d95835183

There is a lot of content shift after the skeleton disappears. I found that adding this spacer in the `SmallSkeleton` component fixes the problem.

```suggestion
const SmallSkeleton = () => (
  <>
    <VSpacer size={8} />
    <Skeleton width={178} height={16} radius={8} />
  </>
);
```

Reply:
Increasing the height does solve the problem, but it shows the content very close to the labels, which is not visually beatifull.